lib/src/repo.rs

```diff
@@ -1108,6 +1108,8 @@ pub struct RewriteRootCommit;
 /// Error from attempts to edit a commit
 #[derive(Debug, Error)]
 pub enum EditCommitError {
+    #[error("Current working-copy commit not found: {0}")]
+    WorkingCopyCommitNotFound(#[from] BackendError),
```

Comment:
Nit: I would add `.map_err()` instead of `#[from]` because it's unclear for me that any `BackendError` for `EditCommitError` occurs on this specific situation.

Reply:
Good point. Fixed.